bridge/common/protos/output.proto

```diff
@@ -38,6 +38,7 @@ enum EnvironmentType {
   MainNet = 0;
   Rococo = 1;
   Dev = 2;
+  TestnetPaseo = 3;
```

Comment:
nit: Since these protos are only used to communicate between rust and jni I think we can just add Paseo after Mainnet and shift the values